src/utils/hooks/useTimeline/useTimeline.tsx

```diff
@@ -157,15 +163,17 @@ const generateTimelineEvent: {
     strings: LanguageStrings,
     decisionStrings: {
       [key in Decision]: string
-    }
+    },
+    locale: string
   ) => EventOutput
 } = {
   STAGE: () =>
     // Already handled in caller
     ({ eventType: TimelineEventType.Ignore, displayString: '' }),
   STATUS: (event, fullLog, _, __, strings) => getStatusEvent(event, fullLog, strings),
   OUTCOME: (event, _, __, ___, strings) => getOutcomeEvent(event, strings),
-  EXTENSION: (event, _, __, ___, strings) => getExtensionEvent(event, strings),
+  EXTENSION: (event, _, __, ___, strings, _____, locale) =>
```

Comment:
```suggestion
  EXTENSION: (event, _, __, ___, strings, ____, locale) =>
```
Getting a little carried away with your underscores there! 😜

Reply:
: P